### DISCUSSION TITLE
Site that tracks all active jailbreak prompts

### DISCUSSION THREAD

**Discussion by alexalbertt** [LeaderbotX400/chatbot-experiments#3] [Show and tell]
Hi! Recently I made a website that tracks and collects all active jailbreak prompts, similar to what this repo does. Would love to work together and share prompts with each other. You can check out the site here: www.jailbreakchat.com

**Comment by LeaderbotX400** [comment]
This sounds cool! Feel free to make pull requests for this repo if you want to add some, I would be happy yo lend a hand if you need one.
